### PR TITLE
Enable syntax highlighting in code blocks

### PR DESCRIPTION
**Pull request by alefteris**
Enable JavaScript syntax highlighting in code blocks in README.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ resumeSchema.validate({ name: "Thomas" }, function (err, report) {
 
 More likely
 
-```
+```js
 var fs = require('fs');
 var resumeSchema  = require('resume-schema');
 var resumeObject = JSON.parse(fs.readFileSync('resume.json', 'utf8'));
```
Comment (danh29645) [inline README.md]: Dan=//*hurleY

Comment (PeterDaveHello) [inline README.md]: ?

**Comment by aloisdg** [comment]
@PeterDaveHello I wonder whether @danh29645 is a bot or not.

File: README.md (hunk 1 above)
Comment (aloisdg) [inline README.md]: @PeterDaveHello I wonder whether @danh29645 is a bot or not.

Comment (PeterDaveHello) [inline README.md]: Not sure, is or not, if it/he/she keep doing this(https://github.com/jsonresume/resume-schema/issues/213), I'll report the abuse behavior.

Comment (olivif) [inline README.md]: the account seems to be disabled now, probably a bot.

Comment (PeterDaveHello) [inline README.md]: nice

**Pull request merged**